libs/datamodel/core/src/configuration/mod.rs

```diff
@@ -17,3 +17,12 @@ pub struct StringFromEnvVar {
     pub from_env_var: Option<String>,
     pub value: String,
 }
+
+impl StringFromEnvVar {
+    pub fn plain_value(value: String) -> StringFromEnvVar {
```

Comment:
What does "plain value" mean in this context?

Reply:
function is unused. Will remove.